Adding videos of how to create projects within Organizations, and adding  the method to workflow (#634)
opengisch/QField-docs: documentation/get-started/tutorials/advanced-setup-qfc.en.md

> How to create a project in an organization:

Reviewer: ```suggestion
There are several ways in which you can create and upload a project to QField.
- Using QFieldSync
- Using QFieldCloud 
- Change of ownership
```

!!! workflow

>     ### Option 1: Using QField Sync

Reviewer: ```suggestion
    ### **Option 1: Using QFieldSync**
```

    5. In QGIS open QFieldSync and you will see the new project listed, click on "Edit Selected Cloud Project".

        ![QFieldSync](../../assets/images/project_organization_05_qfield_sync.png)

    6. Choose the folder where you want to save the project.

        ![Selecting folder](../../assets/images/project_organization_06_selecting_folder.png)

    7. In the selected folder, you can either paste an already worked-on project or save a new one.

        ![Copy to project folder](../../assets/images/project_organization_07_copy_project_to_folder.png)

    8. Once the folder contains the project, you can synchronize it.

        ![QFieldSync Overview](../../assets/images/project_organization_08_qfield_sync_overview.png)

    9. Finally, push the changes to the cloud.

        ![Pushing changes to QFieldCloud](../../assets/images/project_organization_09_pushing_changes_to_cloud.png)

    10. You can verify that the files are present in the Organization project.

        ![Project files over view in QFieldCloud](../../assets/images/project_organization_10_files_overview_in_cloud.png)

>     ### Option 3: Changing the Ownership of a project

Reviewer: ```suggestion
    ### **Option 3: Changing the Ownership of a project**
```
